plugins/mac-capture/mac-display-capture.m

```diff
@@ -602,11 +446,26 @@ static bool switch_crop_mode(obs_properties_t *props, obs_property_t *p,
 		props, "display", obs_module_text("DisplayCapture.Display"),
 		OBS_COMBO_TYPE_LIST, OBS_COMBO_FORMAT_INT);
 
-	for (unsigned i = 0; i < [NSScreen screens].count; i++) {
-		char buf[10];
-		sprintf(buf, "%u", i);
-		obs_property_list_add_int(list, buf, i);
-	}
+	[[NSScreen screens] enumerateObjectsUsingBlock:^(
+				    NSScreen *_Nonnull screen, NSUInteger index,
+				    BOOL *_Nonnull stop
+				    __attribute__((unused))) {
+		char dimension_buffer[4][12];
+		char name_buffer[256];
+		sprintf(dimension_buffer[0], "%u",
+			(uint32_t)[screen frame].size.width);
+		sprintf(dimension_buffer[1], "%u",
+			(uint32_t)[screen frame].size.height);
+		sprintf(dimension_buffer[2], "%d",
+			(int32_t)[screen frame].origin.x);
+		sprintf(dimension_buffer[3], "%d",
+			(int32_t)[screen frame].origin.y);
+		sprintf(name_buffer, "%.200s: %.12sx%.12s @ %.12s,%.12s",
+			[[screen localizedName] UTF8String],
```

Comment:
According to the header `localizedName` is only available in macOS 10.15+:

```
@property (readonly, copy) NSString *localizedName API_AVAILABLE(macos(10.15));
```

If we still want to support building(!) OBS on older platforms, we'd need to throw a preprocessor check around this.

Reply:
Resolved in the latest